readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -170,24 +170,18 @@ def test_get_raw_log(self):
 
         api_user = get(User, user='test', password='test')
         client.force_authenticate(user=api_user)
-        resp = client.get('/api/v2/build/{0}/log.txt'.format(build.pk))
-        self.assertEqual(resp.status_code, 200)
-        resp = client.get(
-            '/api/v2/build/{0}/log/'.format(build.pk),
-            format='txt'
-        )
+        resp = client.get('/api/v2/build/{0}.txt'.format(build.pk))
         self.assertEqual(resp.status_code, 200)
 
-        self.assertEqual(resp.data, build.raw_log)
-        self.assertIn('RTD build information', resp.data)
-        self.assertIn('[rtd-command-info]', resp.data)
+        self.assertIn(b'RTD build information', resp.content)
+        self.assertIn(b'[rtd-command-info]', resp.content)
```

Comment:
Instead of using binary check here, you should decode the content since you want to compare string not data.